test/libraries/AccountAccessParser.t.sol

```diff
@@ -34,6 +52,24 @@ contract AccountAccessParser_decodeAndPrint_Test is Test {
     address constant addr9 = address(9);
     address constant addr10 = address(10);
 
+    function testReproduceBug() public {
```

Comment:
This should be removed but it demonstrates how no state diffs are found when we run this simple code. If you swap `getUniqueWrites` with `getUniqueWritesNew` in the `decodeAndPrint` function. This test _won't_ revert.